### PR TITLE
[14.0][MIG] osi_account_bank_statement

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: osi_account_bank_statement/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Account Bank Reconciliation Extended",
+    "version": "13.0.1.0.1",
```
Comment (mgosai) [inline osi_account_bank_statement/__manifest__.py]: should be v14

**Review by mgosai — CHANGES_REQUESTED** [review]
Some files are having copyright from 2019 while other from 2020
can we have all in sync ? bdw it's already 2021 ;)  
Also, as stated for v13 module. We will be having separate module for all bank rec related patches and will not manipulate this module. 
@bodedra will be handling it

**Comment by dreispt** [comment]
@mgosai The copyright year is good as it is: is should match the work's creation date.
Later work can have later dates.
Older dates should stay like that, to properly claim authorship and protect copyright.

**Comment by smangukiya** [comment]
closing this out because this is not properly migrated to v14 with current PR.
new PR https://github.com/ursais/osi-addons/pull/423

**Pull request closed without merging**